confgenerator/config.go

```diff
@@ -96,11 +86,11 @@ type Logging struct {
 
 type LoggingReceiverFiles struct {
 	IncludePaths []string `yaml:"include_paths,omitempty"`
-	ExcludePaths []string `yaml:"exclude_paths,omitempty"` // optional
+	ExcludePaths []string `yaml:"exclude_paths,omitempty" config:"optional"`
```

Comment:
Is this named `config` because in the future we expect this to grow as a list? Otherwise `presence:"optional"` sounds more accurate.

Reply:
Yes, this will eventually turn into a list. "config" is the category, or namespace for this tag. I don't think we should invent multiple categories here…

Reply:
I would greatly prefer not reinventing the wheel here; there's a convention to use `validate:"required"` (see https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Required)

My hope is that we can adopt that validation library and drop most/all of our validation code.

Reply:
Done (in https://github.com/GoogleCloudPlatform/ops-agent/pull/141/commits/ef84963586d87f199167e3835b61e35381888bc6).